bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -134,7 +136,9 @@ private void initialize(ServerConfiguration conf, ZooKeeper zkc)
             throws UnavailableException {
         try {
             LedgerManagerFactory ledgerManagerFactory = LedgerManagerFactory
```

Comment:
I would suggest getting layout manager from registration  client, rather than construct zk layout manager directly.

Reply:
Thanks, will change it.